settings.js

```diff
@@ -10,8 +10,7 @@ const settings = {
   minimalTimer: process.env.NODE_ENV.includes('production') ? process.env.MIN_TIMER : process.env.TEST_MIN_TIMER,
   maximumTimer: process.env.NODE_ENV.includes('production') ? process.env.MAX_TIMER : process.env.TEST_MAX_TIMER,
   botOperators: process.env.DISCORD_BOT_OPERATORS_ID.replace(' ', '').split(','),
-  mongoDBUri: process.env.MONGODB_URI,
-  JAWSDB_URL: process.env.MONGODB_URI,
+  mongoDBUri: process.env.MONGODB_URI ? process.env.JAWSDB_URL,
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator

`condition ? set this if condition is true : set this if false`

same as if, else. there is no else condition here, it should be 
`mongoDBUri: process.env.MONGODB_URI ? process.env.MONGODB_URI : process.env.JAWSDB_URL,`

Reply:
Thanks for that, i have no experience in javascript what so ever, so i tried to assume from what was already defined.
After testing, sadly the uri start with "mysql:// and must begin with "mongodb://".

Is it preferd to change this condition to also accept mysql? That way, users don't have to copy the given uri, change it, and then paste it into the new defined mongoDBUri variable.

I would need some help if this change is accepted, due to my lack of knowledge.

Reply:
We're using https://mongoosejs.com/ mongoose to connect and build queries for our database, if its not mongodb I dont think it would work. In order to support multiple dbs would be to implement two query builders that would determine which one to use judging by the connection URI or set by the user.

I have yet to find a querybuilder that builds for both noSQL and SQL :/

I dont know how you were able to connect to your DB with MYSQL hehe

Reply:
Neh i was stupid. I accidently used Jawsdb, what is mysql based. Using Mlabs mongodb, makes the bot work. Hence that the logging above shows no problems.

Suggest this discussion is resolved. Answer: Use mongodb and not mysql.